frontend/src/container/AlertHistory/Timeline/Graph/Graph.tsx

```diff
@@ -66,6 +73,40 @@ function HorizontalTimelineGraph({
 					label: 'States',
 				},
 			],
+			hooks: {
+				setSelect: [
+					(self): void => {
+						const selection = self.select;
+						if (selection) {
+							const startTime = self.posToVal(selection.left, 'x');
+							const endTime = self.posToVal(selection.left + selection.width, 'x');
```

Comment:
i hope we have handled the back navigation as well ? example -> if we move from timerange A to B then on moving back the time is correctly updated ?

Reply:
Thanks for pointing this out, @vikrantgupta25, I fixed this issue :slightly_smiling_face: